Mathlib/Algebra/Lie/Weights/Killing.lean

```diff
@@ -370,6 +370,13 @@ lemma root_apply_coroot {α : Weight K H L} (hα : α.IsNonZero) :
   rw [← Weight.coe_coe]
   simpa [coroot] using inv_mul_cancel (root_apply_cartanEquivDual_symm_ne_zero hα)
 
+@[simp] lemma traceForm_coroot (α : Weight K H L) (x : H) :
```

Comment:
It's not clear to me that this should be in the global simp set.
But if it's obvious to you, then I'll defer to your judgment.
Otherwise I suggest leaving it out for now.

Reply:
Good point; on reflection I don't think this should be `simp`.